js-miniapp-sdk/README.md

```diff
@@ -71,44 +71,38 @@ miniApp.getUniqueId()
 
-Note that there are two types of permissions:
-- **Device permissions:** These permissions are for accessing device features, and they will display a platform-specific dialog which is controlled by the Android or iOS operating system. Device permissions can only be requested one at a time.
-- **Custom permissions:** These permissions are related to accessing the User's data which the Host App controls, and the Host App will display a custom permission dialog. Multiple permissions can be requested at once.
+There are two types of permissions:
+- [**Custom permissions:**](#custom-permissions) Access User data or device features which the Host App controls. Displays Host App's custom permission dialog.
+- [**Device permissions:**](#device-permissions) Access device features. Displays Android/iOS platform permission dialog.
 
-| Permission | Type | Method | Description |
-| --- | --- | --- | --- |
-| Location | Custom & Device | `requestLocationPermission()` | Custom and device permission respectively for location. Currently supported types are `CustomPermissionName.LOCATION` & `DevicePermission.LOCATION`.<br>Location data can be accessible via geolocation request (`navigator.geolocation`). |
-| User Data | Custom | `requestCustomPermissions()` | Custom permissions for User Data. Currently supported types are `CustomPermissionName.USER_NAME`, `CustomPermissionName.PROFILE_PHOTO`, and `CustomPermissionName.CONTACT_LIST`. These permissions should be requested before you attempt to access the User's data. |
+#### Custom Permissions
 
-#### Usage example
+**API:** [MiniAppFeatures.requestCustomPermissions](api/interfaces/miniappfeatures.md#requestcustompermissions),
+[CustomPermissionName](api/enums/custompermissionname.md),
+[CustomPermissionStatus](api/enums/custompermissionstatus.md)
 
-Simply call available permission request methods from `miniApp`.
+These permissions are related to accessing the User data or device features which the Host App controls, and the Host App will display a custom permission dialog. Multiple permissions can be requested at once. These permissions should be requested before you attempt to access the User's data or certain device features.
 
-```javascript
-// Location Permission
-import miniApp from 'js-miniapp-sdk';
-miniApp.requestLocationPermission('This description will be shown to the user.')
-	.then(success => {
-		console.log(success); // Allowed.
-	}).catch(error => {
-		console.error(error); // Permission is not granted due to many circumstances.
-    });
-```
+These permissions are requested using the [MiniAppFeatures.requestCustomPermissions](api/interfaces/miniappfeatures.md#requestcustompermissions) method.
+
+| Permission Name | Description |
+| ---- | ---- | ---- |
```

Comment:
One minor issue: Maybe this part is broken? We can use `| ---- | ---- |`

Reply:
Thanks, I updated this.